### PR TITLE
Add new company - Kyndryl

### PR DESCRIPTION
**Pull request by zoltanvin**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -384,6 +384,7 @@ Name | Website | Region
 [Kentik](/company-profiles/kentik.md) | https://www.kentik.com/careers | USA, Europe, APAC
 [Khan Academy](/company-profiles/khan-academy.md) | https://www.khanacademy.org/ | USA
 [KickBack Rewards Systems](/company-profiles/kickback-rewards-systems.md) | https://careers.kickbacksystems.com | USA
+[Kyndryl](/company-profiles/kyndryl.md) | https://www.kyndryl.com/ | Worldwide
```
Comment (KieranRobson) [inline README.md:387]: This should be above `[Kuali](/company-profiles/kuali.md) | https://kuali.co/ | USA`

File: company-profiles/company-profiles/kyndryl.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+## Remote status
+
+5 days work week with flexible hours and WFH policy.
+
+## Region
+
+Worldwide
+
+## Company technologies
```
Comment (KieranRobson) [inline company-profiles/company-profiles/kyndryl.md:19]: This list could be smaller as its long to scroll down, maybe just list the ain technologies or place them into categories?

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Thanks for your additions! A couple of changes to be made!

File: README.md (hunk 1 above)
Comment (zoltanvin) [reply README.md]: I fixed it. Should be below shouldn't it?

Comment (KieranRobson) [reply README.md]: @zoltanvin Woops, yes!

**Comment by zoltanvin** [comment]
Thanks for the input, comments fixed. Please check again

**Comment by KieranRobson** [comment]
The web logs do not load for me so I cant see why checks are failing although everything looks to be right

**Review by KieranRobson — APPROVED** [review]
Using https://github.com/remoteintech/remote-jobs/pull/1609/files as a base, everything looks to be right, just the logs don't seem to be loading

**Comment by zoltanvin** [comment]
Closed bc of inactivity

**Pull request closed without merging**